### PR TITLE
cms_form: handle hidden input automatically

### PR DESCRIPTION
**Pull request by simahawk**
You can now specify `_form_fields_hidden = ('foo', )`
to get hidden inputs. All fields declared here
will be rendered as `<input type="hidden" />`.

### CODE REVIEW

File: cms_form/models/cms_form_mixin.py [hunk 1]
```diff
@@ -214,10 +216,13 @@ def form_model(self):
         # queue_job tries to read properties. Be defensive.
         return self.env.get(self._form_model)
 
-    def form_fields(self):
+    def form_fields(self, hidden=None):
```
Comment (grindtildeath) [inline cms_form/models/cms_form_mixin.py:219]: It took me quite a while to understand the use of the parameter and its effect. Still feels quite weird to me to get 3 different results using True, False or None for what looks like a boolean field. IMO a docstring is needed in such case.

**Review by grindtildeath — APPROVED** [review]
Approved as the code looks ok, but please address the comment below :wink:

File: cms_form/models/cms_form_mixin.py (hunk 1 above)
Comment (simahawk) [reply cms_form/models/cms_form_mixin.py:219]: right, forgot the docstring!

**Review by tschanzt — APPROVED** [review]

File: cms_form/models/cms_form_mixin.py (hunk 1 above)
Comment (simahawk) [reply cms_form/models/cms_form_mixin.py:219]: added

**Review by ghost — APPROVED** [review]

**Pull request merged**